Microsoft.Toolkit.Wpf.UI.XamlHost/WindowsXamlHostBase.cs

```diff
@@ -6,55 +6,44 @@
 using System.Runtime.InteropServices;
 using System.Windows.Interop;
 using Microsoft.Toolkit.Win32.UI.XamlHost;
-using Windows.UI.Xaml;
-using Windows.UI.Xaml.Media;
-using windows = Windows;
+using WUX = Windows.UI.Xaml;
 
 namespace Microsoft.Toolkit.Wpf.UI.XamlHost
 {
     /// <summary>
     /// WindowsXamlHost control hosts UWP XAML content inside the Windows Presentation Foundation
     /// </summary>
-    public abstract partial class WindowsXamlHostBase : HwndHost
+    abstract partial class WindowsXamlHostBase : HwndHost
     {
         /// <summary>
         /// UWP XAML Application instance and root UWP XamlMetadataProvider.  Custom implementation required to
         /// probe at runtime for custom UWP XAML type information.  This must be created before
         /// creating any DesktopWindowXamlSource instances if custom UWP XAML types are required.
         /// </summary>
-        private readonly windows.UI.Xaml.Application _application;
+        /// <remarks>
+        /// Windows.UI.Xaml.Application object is required for loading custom control metadata.  If a custom
```

Comment:
This comment could be ammended now that a custom metadata provider can be added at any time, as long as it is added before the custom type it provides metadata for is instantiated.  There doesn't seem to be any reason to create a custom Xaml.Application object except to provide an implementation of the metadata provider.   Also, WindowsXamlManager would still need to be called if the application needed, for some reason, to create XAML objects before creating DesktopWindowXamlSource.